app/assets/stylesheets/top_bar.scss

```diff
@@ -0,0 +1,38 @@
+.has-submenu > a {
+  border-left: 1px solid white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.